packages/expect/src/toThrowMatchers.ts

```diff
@@ -447,4 +458,26 @@ const formatStack = (thrown: Thrown | null) =>
         },
       );
 
+const _createMessageAndCause = (error: Error): string => {
+  if (error.cause instanceof Error) {
+    return `{ message: ${error.message}, cause: ${_createMessageAndCause(
+      error.cause,
+    )}}`;
+  } else {
+    return `{ message: ${error.message} }`;
+  }
+};
+
+const createMessageAndCause = (error: Error) => {
+  if (error.cause instanceof Error) {
+    return _createMessageAndCause(error);
+  } else {
+    return error.message;
+  }
+};
```

Comment:
why are these 2 functions?

Reply:
Currently, if `{ message: 'foo' }` is given, the standard output is `"Expected message: foo"`.

To maintain this current behavior, I separated the functions so that only error.message is returned in the first loop.

Reply:
`_createMessageAndCause` is only called from within itself or `createMessageAndCause`, can't they be combined?